test/extensions/transport_sockets/tls/integration/ssl_integration_test.cc

```diff
@@ -87,6 +89,65 @@ void SslIntegrationTestBase::checkStats() {
   counter->reset();
 }
 
+class SslKeyLogTest : public SslIntegrationTest {
+public:
+  SslKeyLogTest(){};
+  void setLocalFilter() {
+    keylog_local_ = true;
+    keylog_remote_ = false;
+    keylog_negative_ = false;
+  }
+  void setRemoteFilter() {
+    keylog_remote_ = true;
+    keylog_local_ = false;
+    keylog_negative_ = false;
+  }
+  void setBothLocalAndRemoteFilter() {
+    keylog_local_ = true;
+    keylog_remote_ = true;
+    keylog_negative_ = false;
+  }
+  void setNegative() {
+    keylog_local_ = true;
+    keylog_remote_ = true;
+    keylog_negative_ = true;
+  }
+  void logCheck() {
+    std::string log = waitForAccessLog(TestEnvironment::temporaryPath("tlskey.log"));
+    if (server_tlsv1_3_) {
```

Comment:
In both cases, please add a check that the line ends with `\n`, and only a single `\n`.

Reply:
What's the reason for that check for \n? In my opinion, the log is printed by boringSSL, there is no need to check that.